AzureCalling/app/src/main/res/color-night/button_filled.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We already have button_filled.xml in drawable. would it be possible to create only only button_filled.xml in our code and use color from colors.xml & colors.xml(night)?
For example:

button_filled.xml
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:color="@color/color1" android:state_enabled="true" />
    <item android:color="@color/color2" android:state_enabled="false" />
</selector>

colors.xml (night)
<resources>
    <color name="color2">@android:color/holo_orange_dark</color>
    <color name="color1">@android:color/holo_blue_bright</color>
</resources>

colors.xml
<color name="color2">@android:color/holo_red_dark</color>
<color name="color1">@android:color/holo_green_light</color>

Dark Mode:
![Screen Shot 2021-04-09 at 10 32 32 PM](https://user-images.githubusercontent.com/73618019/114259539-7b1a0380-9983-11eb-830a-fe85145d8f60.png)

Light Mode:
![Screen Shot 2021-04-09 at 10 33 22 PM](https://user-images.githubusercontent.com/73618019/114259554-9d138600-9983-11eb-98cb-e38e367f52a0.png)

We can follow above for others controls.

Reply:
Discussed offline:
We will re-use already defined accentColor but will not define new singe use dedicated for button color.